CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 - jsonrpc: `ChatListItemFetchResult` gets new properties: `summary_preview_image`, `last_message_type` and `last_message_id`
+- New rust only API to send a StatusUpdateItem struct.
```

Comment:
maybe be more explicit here, so the reader of the changelog does not need to search for the name of that added API
```suggestion
- New rust API `send_webxdc_status_update_struct` to send a `StatusUpdateItem`
```